### PR TITLE
[Backport] Removed double occurrence of keywords in sentences.

### PR DESCRIPTION
**Pull request by gelanivishal**
### Original Pull Request
https://github.com/magento/magento2/pull/16182

### Description

Removed double occurrence of keywords "to, the" from sentences.

### Fixed Issues (if relevant)

N/A

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: app/code/Magento/Customer/view/frontend/web/js/password-strength-indicator.js [hunk 1]
```diff
@@ -31,6 +31,11 @@ define([
             this.options.cache.input = $(this.options.passwordSelector, this.element);
             this.options.cache.meter = $(this.options.passwordStrengthMeterSelector, this.element);
             this.options.cache.label = $(this.options.passwordStrengthMeterLabelSelector, this.element);
+
+            // We need to look outside the module for backward compatibility, since someone can already use the module.
+            // @todo Narrow this selector in 2.3 so it doesn't accidentally finds the email field from the
+            // newsletter email field or any other "email" field.
+            this.options.cache.email = $(this.options.formSelector).find(this.options.emailSelector);
```
Comment (sidolov) [inline app/code/Magento/Customer/view/frontend/web/js/password-strength-indicator.js]: Looks like this changes not related to current PR, please, revert.

**Review by sidolov — CHANGES_REQUESTED** [review]

File: app/code/Magento/Customer/view/frontend/web/js/password-strength-indicator.js (hunk 1 above)
Comment (gelanivishal) [reply app/code/Magento/Customer/view/frontend/web/js/password-strength-indicator.js]: Done.

**Comment by gelanivishal** [comment]
@sidolov Please review it.

**Comment by magento-engcom-team** [comment]
Hi @gelanivishal. Thank you for your contribution. 
We will aim to release these changes as part of 2.1.15. 
Please check the release notes for final confirmation.

**Pull request merged**